manifests/bucketeer/charts/batch/templates/cronjob.yaml

```diff
@@ -0,0 +1,283 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "batch-server.fullname" . }}-experiment-status-updater
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ template "batch-server.name" . }}
+    chart: {{ template "batch-server.chart" . }}
+    release: {{ template "batch-server.fullname" . }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: {{ .Values.cronjob.experimentStatusUpdaterSchedule }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          volumes:
+            - name: service-cert-secret
+              secret:
+                secretName: {{ template "service-cert-secret" . }}
+            - name: service-token-secret
+              secret:
+                secretName: {{ template "service-token-secret" . }}
+          containers:
+            - name: experiment-status-updater
+              image: curlimages/curl:8.1.2
+              imagePullPolicy: IfNotPresent
+              volumeMounts:
+                - name: service-cert-secret
+                  mountPath: /usr/local/certs/service
+                  readOnly: true
+                - name: service-token-secret
+                  mountPath: /usr/local/service-token
+                  readOnly: true
+              command:
+                - /bin/sh
+              args:
+                - -c
+                - |
+                  echo "Start experiment-status-updater job."
+                  ENDPOINT="https://web-gateway.{{ .Values.namespace }}.svc.cluster.local:{{ .Values.cronjob.webGatewayPort }}/bucketeer.batch.BatchService/ExecuteBatchJob"
```

Comment:
@Ubisoft-potato 

Unfortunately, this won't work. You need to pass the namespace and the web gateway port via the environment variable.
I think it would be simpler if you create an environment variable called `WEB_GATEWAY_ADDRESS` and put the `https://web-gateway.default.svc.cluster.local:9000`

Same for the other jobs.

Reply:
I have fixed this using env variable.

Reply:
Sorry, my PR is misleading 🙏

Reply:
> Sorry, my PR is misleading 🙏

It's ok, we can fix this 😄.